_meetups/2023-09-13-gogrow.md

```diff
@@ -0,0 +1,17 @@
+---
+collection: meetups
+layout: meetup
+host: gogrow
+
+talks:
+ - title: Ruby on Wasm
+   recording: https://drive.google.com/file/d/1CYuMU3eImN8Pttz6sIge4DMElQT0YidX/preview
+   speakers:
+     - marcos_bellucci
```

Comment:
@delbetu if you have a link to your slides at hand I can add it now. Otherwise you can make a PR afterwards